### DISCUSSION TITLE
Is it possible to view all players with a max ping that is higher than a specific amount in Query?

### DISCUSSION THREAD

**Discussion by Kaludii** [plan-player-analytics/Plan#3254] [Q&A]
As the title states, is it possible to quickly get a list of all the players along with the pings for all the players with the highest ping being more than a specific amount set? I'd like to compare the times where our players' ping spikes to see if there may be an issue with the server. Thanks!

**Comment by AuroraLS3** [comment]
I'll convert this into a suggestion ticket since it is currently not possible.

**Comment by AuroraLS3** [comment]
Please add further details to https://github.com/plan-player-analytics/Plan/issues/3255 if necessary

**Comment by AuroraLS3** [comment]
You can also do this query:
```sql
SELECT uuid, name, MAX(max_ping) FROM plan_ping p
JOIN plan_users u ON u.id=p.user_id
WHERE max_ping > 100 GROUP BY uuid, name
```
